near-contract-standards/src/fungible_token/mod.rs

```diff
@@ -7,5 +7,8 @@ pub mod receiver;
 pub mod resolver;
 pub mod storage_impl;
 
+pub use self::core::FungibleTokenCore;
+pub use crate::storage_management::StorageManagement;
```

Comment:
Why do we need to re-export StorageManagement here?

Reply:
We don't, I kept this consistent with how macros were organised.